kohlivarun5/SmartTix
# Fix resale

Fixes #49 

There is a bug where the wrong ticket gets assigned to other user and original owner can still use the qr. 
Test whole workflow with:
 - Buy tix
 - Get ticket and save image
 - Propose sale
 - Buy as another person
 - Make sure now original image does not work
 - Use remaining tix of orig user
 - Make sure new ticket works for new user
